test/wrappers/challenges_wrapper.js

```diff
@@ -159,6 +159,93 @@ describe('Challenges Wrapper', () => {
     });
   });
 
+  describe('start', () => {
+    const sheltererId = '0xc0ffee';
+    const bundleId = '0xbeef';
+    const fee = '10';
+    let startChallengeStub;
+    let startChallengeSendStub;
+    let contractMock;
+
+    beforeEach(() => {
+      startChallengeStub = sinon.stub();
+      startChallengeSendStub = sinon.stub();
+      startChallengeStub.returns({
+        send: startChallengeSendStub
+      });
+      contractMock = {
+        methods: {
+          start: startChallengeStub
+        }
+      };
+      challengesWrapper = new ChallengesWrapper({}, {}, defaultAddress, true);
```

Comment:
```suggestion
      challengesWrapper = new ChallengesWrapper({}, {}, defaultAddress);
```

I'd prefer to rely on the default value, especially because we plan to get rid of this flag altogether.

Reply:
👍